libc/src/stdlib/atexit.h

```diff
@@ -8,13 +8,8 @@
 
 #ifndef LLVM_LIBC_SRC_STDLIB_ATEXIT_H
 #define LLVM_LIBC_SRC_STDLIB_ATEXIT_H
-
-#include <stddef.h> // For size_t
```

Comment:
this should include the proxy header for `__atexithandler_t` and use it in the function prototype